plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -202,3 +202,15 @@
     comment on column {{ relation }}.{{ adapter.quote(column_name) if column_dict[column_name]['quote'] else column_name }} is {{ escaped_comment }};
   {% endfor %}
 {% endmacro %}
+
+{% macro postgres__get_show_grant_sql(relation) %}
+ {% call statement('get_show_grant_sql', fetch_result=True) -%}
+  select grantee
+  from information_schema.role_table_grants
+      {{ log(grantee, info=True) }}
+      where grantor = current_role
+        and grantee != current_role
```

Comment:
@jtcohen6 where does `current_role` come from here?

Reply:
@dataders This is a value that PostgreSQL makes available to us, in all queries. In other databases, we may need to use dbt's own knowledge of the user/role (`{{ target.user }}` or `{{ target.role }}`) here instead.